src/NRedisStack/Gears/GearsCommandBuilder.cs

```diff
@@ -0,0 +1,77 @@
+using NRedisStack.RedisStackCommands;
+using NRedisStack.Gears.Literals;
+namespace NRedisStack
+{
+
+    public static class GearsCommandBuilder
+    {
+        public static SerializedCommand TFunctionLoad(string libraryCode, string? config = null, bool replace = false)
+        {
+            var args = new List<object>() { "LOAD" };
+
+            if (replace)
+            {
+                args.Add(GearsArgs.REPLACE);
+            }
+
+            if (config != null)
+            {
+                args.Add(GearsArgs.CONFIG);
+                args.Add(config);
+            }
+            args.Add(libraryCode);
+            return new SerializedCommand("TFUNCTION", args);
+        }
+
+        public static SerializedCommand TFunctionDelete(string libraryName)
+        {
+            return new SerializedCommand("TFUNCTION", "DELETE", libraryName);
+        }
+
+        public static SerializedCommand TFunctionList(bool withCode = false, int verbose = 0, string? libraryName = null)
+        {
+            var args = new List<object>() { "LIST" };
+
+            if (withCode)
+            {
+                args.Add("WITHCODE");
+            }
+
+            if (verbose > 0)
```

Comment:
if (verbose > 0 && verbose  < 3) {

} else {
  /// raise an error for an invalid entry
}

Reply:
LGTM, just need to be `verbose < 4` instead of `verbose < 3`